apps/src/lib/levelbuilder/rubrics/LearningGoalItem.jsx

```diff
@@ -27,6 +29,20 @@ export default function LearningGoalItem({
     );
   };
 
+  const handleKeyConceptFocus = event => {
+    if (exisitingLearningGoalData.aiEnabled && !canEditLearningGoalName) {
+      if (
+        confirm(
+          'Please contact the teacher tools team before changing the name of any learning goal that uses AI assessment.'
```

Comment:
super tiny nit: please change "learning goal" to "Key Concept", since that's what it is shown as in the levelbuilder UI. or if we are confident it should be called learning goal, perhaps we should change the "Key Concept" string to "Learning Goal"